.github/workflows/release.yml

```diff
@@ -31,3 +35,28 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           VERSION: ${{ github.ref }}
+  publish:
```

Comment:
If you do it like this, this makes `build` almost obsolete and increase overall runtime.
I will make a proposal to solve this. If you don't like it, simply revert it

Reply:
I agree with you and probably we should put publishing call just after tagging, shouldn't we?

I followed logic, that version is given from a tag, and we put a tag _after_ build. So if we do a publish before tagging, an old version will be published.

Reply:
When  I understand right:
No, you do a `git tag -a 1.x.s`omething` -m "something" && git push --tags` this creates the tag, and **after** this the CI starts to run